### PR TITLE
Add content icon

### PR DESCRIPTION
**Pull request by leileu**
### WHY are these changes introduced?

Updating the content icon on the left nav in admin as part of the metaobject release.

### WHAT is this pull request doing?

Adding "Content_minor" icon to polaris library (Both in code and in figma library)

### CODE REVIEW

**Comment by leileu** [comment]
@sainihas This is the PR for the content icon

**Comment by leileu** [comment]
@kyledurand @sarahill @heyjoethomas Could you all kindly help me approve this PR? Thank you!

File: .changeset/blue-turkeys-impress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': minor
+---
+
+Adding content minor icon for left nav in admin
```
Comment (kyledurand) [inline .changeset/blue-turkeys-impress.md:5]:
```suggestion
Added ContentMinor icon```

File: polaris-icons/icons/ContentMinor.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: Content
+set: Minor
```
Comment (kyledurand) [inline polaris-icons/icons/ContentMinor.yml:2]:
```suggestion
set: minor
```

**Review by kyledurand — APPROVED** [review]
Looks good just a couple of minor tweaks

**Review by sarahill — APPROVED** [review]

File: .changeset/blue-turkeys-impress.md (hunk 1 above)
Comment (leileu) [reply .changeset/blue-turkeys-impress.md:5]: Thank you!

**Pull request merged**